net/mlvpn/files/mlvpn-updown

```diff
@@ -0,0 +1,115 @@
+#!/bin/sh
+
+# up/down script for MLVPN.
+#
+# MLVPN calls this script with at least 2 arguments:
+# $1 : interface name
+# $2 : "command"
+# command can be:
+#    - "tuntap_up"
+#    - "tuntap_down"
+#    - "rtun_up" $3 : tunnel name
+#    - "rtun_down" $3 : tunnel name
+# tuntap_up is called when at least one tunnel is up
+# tuntap_down is called when at every tunnel is down
+# rtun_up is called when successfully connected
+# rtun_down is called when disconnected
+
+# Environment variables are set by mlvpn, reflecting
+# settings in mlvpn.conf
+# IP4=
+# IP6=
+# IP4_GATEWAY=
+# IP6_GATEWAY=
+# IP4_ROUTES=
+# IP6_ROUTES=
+# MTU=
+# DEVICE=
+
+DEVICE="$1"
+STATUS="$2"
+
+[ -z "$STATUS" ] || [ -z "$DEVICE" ] || [ -z "$MTU" ] && exit 1
+
+unamestr=$(uname)
+
+link_up()
+{
+    if [ "$unamestr" = "Linux" ]; then
+        ip link set dev "$DEVICE" mtu "$MTU" up
+        if [ ! -z "$IP4" ]; then
```

Comment:
shellcheck throws numerous instances of:

SC2236: Use -n instead of ! -z.

https://github.com/koalaman/shellcheck/wiki/SC2236